src/main/service/sjpFilterService.ts

```diff
@@ -66,6 +71,16 @@ export class SjpFilterService {
             });
         });
 
+        const hasLondonPostalAreaCode = this.checkForLondonPostalAreaCodes(londonPostalAreaCodes, postalAreaCodes);
+
+        if (hasLondonPostalAreaCode) {
+            filterStructure.postcodes.push({
+                value: 'The City of London',
+                text: 'The City of London',
+                checked: filterValues.includes('The City of London'),
```

Comment:
"The City of London' is used many times in this class...might be better to define a constant at the top